workflow/applications/gfs_forecast_only.py

```diff
@@ -117,6 +117,9 @@ def get_task_names(self):
 
         if self.do_gempak:
             tasks += ['gempak']
+            tasks += ['gempakmeta']
+            tasks += ['gempakncdcupapgif']
+            tasks += ['gempakpgrb2spec']
```

Comment:
This is fine now because we are considering `forecast-only` to be used only with `RUN=gfs`.  In the past, one could have used `forecast-only` with `RUN=gdas`.  It will take some effort to reinstate that capability.
This is just a note; no action required.